File: app/src/test/java/com/example/borysgurtovyi/codinginterviewprep/MoveZeroesTest.java
package com.example.borysgurtovyi.codinginterviewprep;

import org.junit.Test;

import java.util.Arrays;

/**
 * Created by borysgurtovyi on 2/12/18.
 */

public class MoveZeroesTest {

    @Test
    public void testMoveZeroes() {
        MoveZeroes moveZeroes = new MoveZeroes();

        int[] nums = {0, 1, 0, 3, 12};
        moveZeroes.moveZeroes(nums);

        System.out.println(Arrays.toString(nums));

    }

}

File: app/src/test/java/com/example/borysgurtovyi/codinginterviewprep/MoveZeroesTest.kt
package com.example.borysgurtovyi.codinginterviewprep

import org.junit.Test

/**
 * Created by lisa on 15.02.2018.
 */
class MoveZeroesTest {

    @Test
    fun moveZeroesTest(){
        val moveZeroes = MoveZeroes()
        val nums: Array<Int> = arrayOf(0,1,0,3,12)
        moveZeroes.moveZeroes(nums)
        println(nums.asList())
    }
}

